### ISSUE TITLE
Consider making tabs <section> into <div> 

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#1339]
On encountering [`<section>` in tabs](https://github.com/alphagov/govuk-frontend/blob/fea29aaa833d5f8233c71e0d076923a697e752cd/src/components/tabs/template.njk#L26), HTML validator gives the following warning: 
> Section lacks heading. Consider using h2-h6 elements to add identifying headings to all sections.

if it doesn't contain a heading.

According to the [HTML specs](https://www.w3.org/TR/html52/sections.html#the-section-element)
> Each section should be identified, typically by including a heading (h1-h6 element) as a child of the section element.

We should consider making this element a `<div>`.
